Interni/StudioDiFattibilità/components/fattibilità-c1.tex

```diff
@@ -0,0 +1,72 @@
+\section{Capitolato C1}
+
+\subsection{Informazioni generali}
+\begin{itemize}
+\item \textbf{Nome:} BlockCOVID
+\item \textbf{Proponente:} Imola Informatica
+\item \textbf{Committenti:} \committenti{}
+\end{itemize}
+
+\subsection{Descrizione del capitolato}
+Il capitolato vuole realizzare un sistema che permetta di tutelare la salute e la sicurezza dei lavoratori dal rischio contagio e garantire la salubrità dell'ambiente di lavoro. E' chiesto dunque di realizzare un apparato che soddisfi:
+\begin{itemize}
+\item il tracciamento immutabile e certificato delle presenze in tempo reale nelle postazioni di lavoro di un laboratorio informatico, contrassegnate tramite dei tag RFID;
+\item il tracciamento immutabile e certificato della pulizia delle postazioni.
+\end{itemize}
+\subsection{Finalità del progetto}
+Il prodotto e' composto dalle seguenti parti:
+\begin{itemize}
+\item Il server centrale, gestisce le varie stanze e postazioni, deve poter:
+\begin{itemize}
+\item sapere in ogni momento se la postazione è occupata, prenotata oppure da pulire (condizioni per cui
+lo studente/dipendente non può utilizzarla);
+\item controllare quali postazioni sono prenotate (da chi) e bloccare le prenotazioni per una determinata stanza;
+\item prevedere una tracciatura autenticata e tutti i cambiamenti di stato relativi alla pulizia della
+postazione, nonché le informazioni su chi ha igienizzato la postazione, devono essere salvate su
+memoria immutabile e certificata;
+\item Deve essere possibile prenotare una postazione con granularità di 1 ora;
+\item Il motore di calcolo che si occupa solamente di trovare il migliore percorso da far percorrere alla singola unità, in base a diversi algoritmi di ricerca operativa per l'ottimizzazione del percorso.
+\end{itemize}
+\item GUI lato amministratore:
```

Comment:
Questo punto vale anche per il prossimo elemento della lista "GUI lato utente [..]".
Questi due elementi vanno a stonare con il primo punto, mentre nel primo ci si riferisce al server con "Il server"
in questi punti ci si riferisce senza aggettivo